src/Elastic.Apm/Api/Tracer.cs

```diff
@@ -3,50 +3,39 @@
 using System.Reflection;
 using System.Threading;
 using System.Threading.Tasks;
+using Elastic.Apm.Config;
 using Elastic.Apm.Helpers;
 using Elastic.Apm.Logging;
 using Elastic.Apm.Model.Payload;
+using Elastic.Apm.Report;
 
 namespace Elastic.Apm.Api
 {
 	internal class Tracer : ITracer
 	{
 		private static AbstractLogger _publicTracerLogger;
 
-		public ITransaction CurrentTransaction
-			=> TransactionContainer.Transactions.Value;
+		public ITransaction CurrentTransaction => TransactionContainer.Transactions.Value;
 
-		public static AbstractLogger PublicTracerLogger => _publicTracerLogger ?? (_publicTracerLogger = Agent.CreateLogger("AgentAPI"));
+		public static AbstractLogger PublicTracerLogger => _publicTracerLogger;
```

Comment:
I think this remains `null`. This would be a logger that is used by the public API.

Reply:
I removed this property all together in the subsequent commits, Tracer now gets its logger passed in like the rest of the components 👍

Reply:
👍